build-tools/automation/yaml-templates/fail-on-issue.yaml

```diff
@@ -1,5 +1,5 @@
 parameters:
-  condition: succeeded()
+  condition: succeededOrFailed()
```

Comment:
If the job has already been marked as failed, is there any value in running this to mark it as failed again?

Reply:
I think it's probably safe to swap this condition back, but it should also be harmless.